pkg/config/config.go

```diff
@@ -185,6 +185,9 @@ func initConfig(config Config) {
 	// Defaults to safe YAML methods in base and custom checks.
 	config.BindEnvAndSetDefault("disable_unsafe_yaml", true)
 
+	// Yaml keys which values are stripped from flare
+	config.BindEnvAndSetDefault("yaml_stripped_keys", []string{"community_string", "authKey", "privKey"})
```

Comment:
Could we rename this to something like `flare_stripped_keys`, I don't think `yaml_stripped_keys` is very clear on it's usage.